src/sass/components/_section--promotion.scss

```diff
@@ -0,0 +1,255 @@
+.section--promotion {
+  padding: 0 0 1.875rem 0;
+
+  .promotion {
+    height: 530px;
+    padding: 0 30px;
+
+    .top-bar {
+      background: $text-color;
+      height: 50px;
+      color: $top-bar-color;
+      text-transform: uppercase;
+      font-size: 24px;
+      font-weight: 500;
+      padding: 7px 30px;
+      position: relative;
+
+      .dots {
+        position: relative;
+        right: -120px;
+
+        ul {
+          margin: 0;
+          padding-left: 1rem;
+          list-style: none;
+          width: 90px;
+          transform: translateY(15px);
+          left: 90px;
+          bottom: 20px;
+
+          li {
+            width: 15px;
+            height: 15px;
+            background: $promotion-dots-color;
+            margin: 4px;
+            border-radius: 50%;
+
+            &:hover,
+            &.active {
+              background: $primary;
+            }
+          }
+        }
+      }
+    }
+
+    .photo {
+      background-size: cover;
+      background-position: center;
+      height: 300px;
+
+      .hover {
```

Comment:
we have a `.hover:hover` selector here. can it be named more properly like `layer` or `add-to-card`?